README.md

```diff
@@ -25,21 +27,28 @@ The following example shows a typical topology for a Key Protect instance:
 │   ├── root-key-ocp-cluster-...
 ```
 
+## Using HPCS instead of Key Protect
```

Comment:
@MatthewLemmond You might be able to reuse this text from the CBR module (in reverse): 

The module supports setting up rules for Key Protect and Hyper Protect Crypto Services. By default, the module sets rules for Hyper Protect Crypto Services, but this can be modified to use Key Protect, Hyper Protect, or both Key Protect and Hyper Protect Crypto Services using the input variable `kms_service_targeted_by_prewired_rules`

https://github.com/terraform-ibm-modules/terraform-ibm-cbr/tree/main/modules/fscloud

Reply:
@SirSpidey I've updated line 32 in the readme to more closely reflect this block from the CBR module, let me know if this looks good to you